ansible/site.yml

```diff
@@ -87,7 +87,7 @@
     - name: Disable cray-ifconf
```

Comment:
Can we remove this task instead of a task that says "Disable cray-ifconf" but then it sets `enable: yes`? We can leave it to COS to configure `cray-ifconf`.

Reply:
Removed.